src/lib/esnext.arraybuffer.d.ts

```diff
@@ -0,0 +1,58 @@
+interface ArrayBuffer {
+	/**
+	 * Read-only. The maximum length that this ArrayBuffer can be resized to (in bytes).
+	 */
+	readonly maxByteLength: number;
+
+	/**
+	 * Read-only. Whether this ArrayBuffer can be resized or not.
+	 */
+	readonly resizable: boolean;
+
+	/**
+	 * Resizes the ArrayBuffer to the specified size (in bytes).
+	 * @param newLength The new length, in bytes, to resize the ArrayBuffer to.
+	 */
+	resize(newLength: number): undefined;
```

Comment:
I would call this `newByteLength` like the proposal does. Then consider deleting the jsdoc.